app/controllers/planning_applications/assessment/requirements_controller.rb

```diff
@@ -31,11 +32,46 @@ def create
         end
       end
 
+      def edit
+        respond_to do |format|
+          format.html
+        end
+      end
+
+      def update
+        @requirement = @planning_application.requirements.find(params[:id])
```

Comment:
You could remove this line and add `update` to the set_requirement hook above, since I think the code is the same.